### ISSUE TITLE
Outline Button outline stroke is too light

### ISSUE THREAD

**Issue by clocksmith** [flutter/flutter#19895]
The material outline button has a stroke that is defined at 12% opacity of the provided color. Even in its best state, where the onSurfaceColor is black and the surfaceColor is white (which happens to be the default), the contrast ratio is only 1.31:1 [1].

[1] http://contrast-ratio.com/#hsla%280%2C0%25%2C0%25%2C.12%29-on-white

**Comment by dnfield** [comment]
It looks like we're not actually doing anything with the color - we have something to check whether the color is dark or not, and use this logic (for an enabled button): `(isDark ? Colors.grey[600] : Colors.grey[200])`

`Colors.grey[600]` is `#FF757575`.
`Colors.grey[200]` is `#FFEEEEEE`.

Can you offer any guidance around more appropriate color(s) to use?

**Comment by dnfield** [comment]
Actually, it looks like `isDark` is not actually calculating the brightness of the color supplied, but rather is user and/or theme supplied.

/cc @HansMuller

**Comment by dnfield** [comment]
Based on offline conversation with @clocksmith - he has a good part of the work needed for this ready and will be putting up a PR for it. There's some question as to whether the color/opacity really needs to be changed that's being worked out in the material spec for right now, but at the very least the current implementation can be improved - the color of the outline isn't actually derived from the color of the button.
